src/common/target.c

```diff
@@ -168,7 +168,6 @@ static const TargetEntry TargetMap[] = {
     {   "sim6502",      TGT_SIM6502     },
     {   "sim65c02",     TGT_SIM65C02    },
     {   "supervision",  TGT_SUPERVISION },
-    {   "vc20",         TGT_VIC20       },
```

Comment:
Well, it's undocumented, so I don't see any compatibility breaks.
What about vic1001 or apple2euro? Simply don't support any explicit aliases.